README.md

```diff
@@ -7,85 +8,72 @@ The snap packaging makes it easy to run the Chip Tool on Linux distributions.
 
 This snap has been tested on amd64/arm64 architectures for WiFi/Ethernet/DNS-SD/BLE/Thread commissioning and control.
 
-## Usage
+Usage instructions can be found in the [documentation](https://canonical-matter.readthedocs-hosted.com/en/latest/how-to/chip-tool-commission-and-control/).
 
-### Setup
+## Development
 
-```bash
-sudo snap install chip-tool
-```
+### Build the snap
 
-When installing from the Snap Store, the following interfaces auto-connect:
-- [`avahi-observe`](https://snapcraft.io/docs/avahi-observe-interface) to allow DNS-SD based discovery.
-- [`bluez`](https://snapcraft.io/docs/bluez-interface) for device discovery over Bluetooth Low Energy (BLE).
+Build locally for the same architecture as the host:
 
-> **Note**  
-> DNS-SD and Bluetooth depend on Avahi and Bluez.
-> To install:
-> - Ubuntu: `sudo apt install bluez avahi-daemon`
-> - Ubuntu Core: `sudo snap install avahi bluez`
+```bash
+snapcraft -v
+```
 
+Build remotely for all supported architectures:
 
-You may connect the [`process-control`](https://snapcraft.io/docs/process-control-interface) interface to allow system-wide process management.
-This is needed to grant Chip Tool access to make [sched_setattr](https://man7.org/linux/man-pages/man2/sched_setattr.2.html) system calls. This may improve the reliability of the commissioning and control operations (see [#8](https://github.com/canonical/chip-tool-snap/issues/8)).
 ```bash
-sudo snap connect chip-tool:process-control
+snapcraft remote-build
 ```
 
-### Commissioning into IP network
-Discover using DNS-SD and pair:
+### Install the built snap
+
+Install the local snap:
+
 ```bash
-chip-tool pairing onnetwork 110 20202021
+sudo snap install --dangerous *.snap
 ```
 
-where:
-
--   `110` is the node id being assigned to the app
--   `20202021` is the pin code set on the app
+### Connect interfaces
 
-### Commissioning into Thread network over BLE
-This depends on an OpenThread Border Router (OTBR) with an active network.
-For guidance on setting that up using the [OTBR snap](https://snapcraft.io/openthread-border-router), refer to [this tutorial](https://github.com/canonical/openthread-border-router-snap/wiki/Commission-and-control-a-Matter-Thread-device-via-the-OTBR-Snap).
+When the snap is installed from the Snap Store, the `avahi-observer` and `bluez` interfaces are connected automatically.
+If you install the snap from a local file, like when you built it yourself, you need to connect these interfaces manually:
```

Comment:
You can download the snap from the store and install it from the local file. Once you acknowledge the assertion, snap can be installed without using dangerous mode and the interfaces auto connect:
```console
$ snap download chip-tool
Fetching snap "chip-tool"
Fetching assertions for "chip-tool"
Install the snap with:
   snap ack chip-tool_96.assert
   snap install chip-tool_96.snap

$ snap ack chip-tool_96.assert 

$ sudo snap install ./chip-tool_96.snap 
chip-tool v1.3.0.0+snap from Canonical IoT Labs installed

$ snap connections chip-tool
Interface        Plug                       Slot            Notes
avahi-observe    chip-tool:avahi-observe    :avahi-observe  -
bluez            chip-tool:bluez            :bluez          -
network          chip-tool:network          :network        -
network-bind     chip-tool:network-bind     :network-bind   -
process-control  chip-tool:process-control  -               -
```

So: 
```suggestion
If you install the locally built snap, you need to connect these interfaces manually:
```